### PR TITLE
Tiny fixes in  tools.

### PR DESCRIPTION
**Pull request by TomFischer**
As titled.

### CODE REVIEW

File: Applications/Utils/MeshEdit/removeMeshElements.cpp [hunk 1]
```diff
@@ -160,6 +160,9 @@ int main (int argc, char* argv[])
 	std::unique_ptr<MeshLib::Mesh const> new_mesh(MeshLib::removeElements(
 	    *mesh, searcher.getSearchedElementIDs(), mesh->getName()));
 
+	if (new_mesh == nullptr)
+		return EXIT_FAILURE;
```
Comment (endJunction) [inline Applications/Utils/MeshEdit/removeMeshElements.cpp]: I'd add a reason for the exit...

**Comment by endJunction** [comment]
:fast_forward:

File: Applications/Utils/MeshEdit/removeMeshElements.cpp (hunk 1 above)
Comment (TomFischer) [inline Applications/Utils/MeshEdit/removeMeshElements.cpp]: The function `removeElements` prints already the reason for the exit. Besides the value `nullptr` is returned in two cases (no element or all elements of the mesh will be removed). This makes a specific message at this place difficult. Of course it is possible to add more logic to distinct the cases and print a reason here.

**Comment by chleh** [comment]
:fast_forward:

File: Applications/Utils/MeshEdit/removeMeshElements.cpp (hunk 1 above)
Comment (endJunction) [inline Applications/Utils/MeshEdit/removeMeshElements.cpp]: OK.

**Pull request merged**